README.md

```diff
@@ -8,15 +8,15 @@
 ## Installation
 
 ```lua
-use{
-  'nvim-telescope/telescope.nvim',
-  requires = {
-    'nvim-telescope/telescope-z.nvim',
-  },
-  config = function()
-    require'telescope'.load_extension'z'
-  end,
-}
+-- packer.nvim:
+use{ 'nvim-telescope/telescope-z.nvim', requires = { 'nvim-telescope/telescope.nvim' } }
+...
+require'telescope'.load_extension'z'
```

Comment:
```suggestion
use{
  'nvim-telescope/telescope.nvim',
  requires = {
    {'nvim-lua/plenary.nvim'},
    {'nvim-lua/popup.nvim'},
    {'nvim-telescope/telescope-z.nvim'},
  },
  config = function()
    require'telescope'.load_extension'z'

    // ... other telescope settings ...
  end,
}
```

With packer.nvim in usual, we use `config` section to write plugin-specific settings.

Reply:
It is also good to add requirements for your paq.nvim example.